stack/base-with-services/before-notebook.d/30_start-rabbitmq.sh

```diff
@@ -6,6 +6,9 @@ RABBITMQ_DATA_DIR="/home/${NB_USER}/.rabbitmq"
 mkdir -p "${RABBITMQ_DATA_DIR}"
 fix-permissions "${RABBITMQ_DATA_DIR}"
```

Comment:
Could you explain what this line does?

Reply:
The [fix-permissions script](https://github.com/jupyter/docker-stacks/blob/main/base-notebook/fix-permissions) ensures that the permissions of the data dir are set such that the user's group has read and write access to it.